<ui version="4.0" >
 <class>MpiCsvExportConfig</class>
 <widget class="QDialog" name="MpiCsvExportConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CSV/TSV export configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="fieldSeparatorBox" >
     <property name="title" >
      <string>Field separator</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="3" >
       <widget class="QRadioButton" name="commaSeparator" >
        <property name="text" >
         <string>Comma (CSV file)</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QRadioButton" name="tabSeparator" >
        <property name="text" >
         <string>Tab (TSV file)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="otherSeparator" >
        <property name="text" >
         <string>Other:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>98</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="otherSeparatorInput" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="inputMask" >
         <string>X; </string>
        </property>
        <property name="text" >
         <string>;</string>
        </property>
        <property name="maxLength" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="variousBox" >
     <property name="title" >
      <string>Various options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="writeHeader" >
        <property name="text" >
         <string>Write attribute names to first line</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>164</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MpiCsvExportConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>221</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel" >
     <x>115</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MpiCsvExportConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>304</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel" >
     <x>279</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
